api/routes/tags.py

```diff
@@ -0,0 +1,55 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from api.api_models.tags import TagCreate
+from api.api_models.user import Tags
+from db.database import get_db
+from sqlalchemy.orm import Session, joinedload
+from db.models.users import Tag, User, UserTag
+
+from utils.oauth2 import get_current_user
+
+
+
+tag_route = APIRouter(tags=["User"], prefix='/users')
+
+
+@tag_route.get("/tags")
+def get_current_user_tags(user=Depends(get_current_user), db:Session = Depends(get_db)):
+    db_query =  db.query(User).filter(User.id == user.id).\
+        options(joinedload(User.tags)).first()   
+    return {"tags": db_query.tags}
+
+@tag_route.post('/tags', response_model=Tags, status_code=status.HTTP_201_CREATED)
+def create_tag(tag: TagCreate, current_user=Depends(get_current_user), db:Session=Depends(get_db)):
+    if not current_user:
+        raise HTTPException(status_code=401, detail='Unauthorized')
+
+    user_tag = db.query(Tag).filter(Tag.name == Tag.name).first()
```

Comment:
`filter(Tag.name == Tag.name).first()`

What's being compared here? I don't get it. The field name is equal to what?

Reply:
my bad, it was supposed to be filter(Tag.name == tag.name).first()

Reply:
both works

Reply:
If it works then cool